media-libs/libdvbpsi/libdvbpsi-1.3.3.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Please update to EAPI 7, and change `DEPEND` to `BDEPEND` to install doc tools natively while cross-compiling.

Reply:
> Please update to EAPI 7, and change DEPEND to BDEPEND to install doc tools natively while cross-compiling.

Done! 
https://github.com/gentoo/gentoo/pull/19050/commits/f97e4c60f924abcfe8207ebe4e52632738ee259e